docs/README.md

```diff
@@ -83,41 +83,57 @@ The Notebooks can run on various compute targets; the ones currently supported o
 
 # Productionalization
 
-Designing and building a production data pipeline for a Predictive Maintenance solution can be a relatively non-trivial task. One of the main reasons being that feature engineering (featurization) requires not only new, but also historic data. For reasons outlined below, this solution was built to support real-time featurization and scoring.
+As it is often the case when dealing with time series, feature engineering (featurization) requires access to historical data in addition to the new data. For that reason, designing and building a production data pipeline for a Predictive Maintenance solution can be a relatively non-trivial task.
+
+Generally, predictions can be made either in real time or on a batch basis. This solution was built to support near real-time featurization and scoring, which means up-to-date predictions are generated almost as soon as new data is available.
 
 ## Model operationalization
 
+When using Azure Machine Learning (currenty, *Internal Preview*), it takes the following steps to operationalize a model:
```

Comment:
This has the same typo as it had when I first reviewed it.  Please check the comments from my previous code review.